virtcontainers/qemu.go

```diff
@@ -446,6 +446,7 @@ func (q *qemu) waitSandbox(timeout int) error {
 	defer func(qemu *qemu) {
 		if q.qmpMonitorCh.qmp != nil {
 			q.qmpMonitorCh.qmp.Shutdown()
+			q.qmpMonitorCh.qmp = nil
```

Comment:
is this specific to the vsock addition?  Can you clarify why this is part of the initial commit?

Reply:
We use the  qmp socket later to hotoplug the vsock device
Because we call Shutdown to close the qmp channel, later when we try to inialize the qmp connection again we dont do it because the qmp instance already exist but with the channel closed leading to fail to use it.

Reply:
@jcvenegas This seems to be an existing bug, can you put this in a separate commit with the explanation you provided above as the commit message.

Reply:
This is being taken care of by https://github.com/kata-containers/runtime/pull/501

Reply:
sweet, not needed anymore.

Reply:
Yeah #501 has been merged, you can simply remove the commit now :) (after rebasing obviously!)